nima/webserver/webserver/src/main/java/io/helidon/nima/webserver/WebServer.java

```diff
@@ -120,10 +122,20 @@ default boolean hasTls() {
      */
     boolean hasTls(String socketName);
 
+    /**
+     * Gets a {@link WebServer} context.
```

Comment:
Do not use verbs in javadocs.
This should describe what it is, such as "Context associated with the webserver, used as a parent for request contexts"